2.7.0/final.md

```diff
@@ -0,0 +1,1034 @@
+# PyTorch 2.7.0 Release Notes
+- [Highlights](#highlights)
+- [Backwards Incompatible Changes](#backwards-incompatible-changes)
+- [Deprecations](#deprecations)
+- [New Features](#new-features)
+- [Improvements](#improvements)
+- [Bug fixes](#bug-fixes)
+- [Performance](#performance)
+- [Documentation](#documentation)
+- [Developers](#developers)
+- [Security](#security)
+
+# Highlights
+TODO @seemethere
+
+For more details about these highlighted features, you can look at the release blogpost.
+Below are the full release notes for this release.
+
+# Backwards Incompatible Changes
+
+### Dropped support for Triton < 2.2.0. Removed Support for CUDA 12.4, Anaconda in CI/CD.
```

Comment:
@albanD moved the platform stuff from deprecations to here--lmk if this is better